src/viur/core/skeleton.py

```diff
@@ -1368,6 +1368,17 @@ def fromSkel(cls, kindName: str, *args: list[str]) -> t.Type[RefSkel]:
         newClass.__boneMap__ = bone_map
         return newClass
 
+    def read(self) -> SkeletonInstance:
+        """
+        Read full skeleton instance of the RefSkel from the database.
+        Can be used in the RelationalBone for reading the full Skeleton from a RefSkel
+        :raise ValueError: If the entry is no longer in the database.
+        """
+        skel = skeletonByKind(self["key"].kind)()
+        if not skel.fromDB(self["key"]):
+            raise ValueError(f"""The key {self["key"]!r} seems to be gone""")
+        return skel
+
```

Comment:
Hello, because there was no change [regarding to my proposal](https://github.com/viur-framework/viur-core/pull/1193#pullrequestreview-2135517669), I would like to suggest this alternative version of `RefSkel.read()`. It also doesn't use the kind name from the key, but the `kindName` from the class, which is safer.

```suggestion
    def read(self, key: t.Optional[db.Key | str | int] = None) -> SkeletonInstance:
        """
        Read full skeleton instance referenced by the RefSkel from the database.

        Can be used for reading the full Skeleton from a RefSkel.
        The `key` parameter also allows to read another, given key from the related kind.

        :raise ValueError: If the entry is no longer in the database.
        """
        skel = skeletonByKind(self.kindName)()

        if not skel.fromDB(key or self["key"]):
            raise ValueError(f"""The key {self["key"]!r} seems to be gone""")

        return skel

```